osbenchmark/resources/default-operations.json.j2

```diff
@@ -0,0 +1,16 @@
+    {
+      "name": "index-append",
+      "operation-type": "bulk",
```

Comment:
This looks good. Just the custom-operations.json.j2 needs to be updated to reflect this structure.